terminus-ui/csv-entry/src/csv-entry.component.ts

```diff
@@ -837,18 +865,18 @@ export class TsCSVEntryComponent implements OnInit, OnDestroy {
    */
   private generateBlob(content: TsCSVFormContents): Blob {
     const prefix = 'data:text/csv;charset=utf-8,';
-    const headers: string = content.headers.join('\t') + '\r\n';
-    const rows: string = content.records.map((v) => {
-      // Encapsulate content with quotes and escape any existing quotes
-      return v.columns.map((column) => column ? `"${column.replace(/"/g, '""')}"` : '').join('\t');
-    }).join('\r\n') + '\r\n';
+    const headers = `${content.headers.join('\t')  }\r\n`;
+    // Encapsulate content with quotes and escape any existing quotes
```

Comment:
This comment is trying to explain what the next line is doing since I don't feel the code is extremely clear at first glance.

Possibly there is a better way to word the comment?